spread.yaml

```diff
@@ -431,9 +427,6 @@ backends:
             - debian-sid-64:
                   username: debian
                   password: debian
-            - centos-7-64:
-                  username: centos
-                  password: centos
```

Comment:
Could this be replaced with centos 9?

Reply:
we are running in centos-9 already

Reply:
Not yet for the qemu backend, as far as I can tell.